auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -635,8 +930,10 @@ public T setProviders(@NonNull List<IdpConfig> idpConfigs) {
          * <p>SmartLock is enabled by default.
          *
          * @param enabled enables smartlock's credential selector and hint selector
+         * @deprecated use the more customizable {@link #setIsSmartLockEnabled(boolean, boolean)}
```

Comment:
I don't see a reason to deprecate this "disable all" convenience version.

Reply:
Agreed, I don't remember why I deprecated it. 😕